client/src/ui/css/jsonview.css

```diff
@@ -0,0 +1,77 @@
+.line {
+	margin: 4px 0;
+	display: flex;
+	justify-content: flex-start;
+}
+
+.caret-icon {
+	width: 18px;
+	text-align: center;
+	cursor: pointer;
+}
+
+.empty-icon {
+	width: 18px;
+}
+
+.json-type {
+	margin-right: 4px;
+	margin-left: 4px;
+}
+
+.json-key {
+	color: #444;
+	margin-right: 4px;
+	margin-left: 4px;
+}
+
+.json-index {
+	margin-right: 4px;
+	margin-left: 4px;
+}
+
+.json-separator {
+
+}
```

Comment:
Let's remove the empty class.

Reply:
Fixed with https://github.com/wso2-extensions/identity-tools-plugin-vscode/pull/7